src/Pelagos/Util/DataStore.php

```diff
@@ -424,4 +410,26 @@ protected function getStoreFileName($datasetId, $type)
         }
         return $storeFileName;
     }
+
+    /**
+     * Get the numeric ID of a POSIX user or group.
+     *
+     * @param string  $name    The user or group name.
+     * @param boolean $isGroup Flag set to true if a group, defaults to user.
+     *
+     * @return string|boolean
+     */
+    protected function getIdFromName(string $name, bool $isGroup = false)
+    {
+        $id = false;
+        if ($isGroup) {
+            // This call returns false on failure to resolve.
+            $obj = posix_getgrnam($name);
```

Comment:
Is ext-posix required to be added to composer.json?

Reply:
No, this is the built-in PHP function.